models/webhook_discord.go

```diff
@@ -0,0 +1,242 @@
+package models
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	"code.gitea.io/git"
+	api "code.gitea.io/sdk/gitea"
+
```

Comment:
remove this empty line.

Reply:
above is other packages on gitea, below is submodule on gitea repo.

Reply:
Got it 👍

Reply:
In that case they should be in the other order.

```
builtin
local package
external package
```

Reply:
@bkcsoft done.